src/sentry/utils/cloudfunctions.py

```diff
@@ -13,6 +13,59 @@
 from google.cloud.pubsub_v1 import PublisherClient
 from google.protobuf.field_mask_pb2 import FieldMask
 
+from sentry.utils import json
+
+WRAPPER_JS = """
+const userFunc = require('./user.js');
+Object.assign(process.env, require('./env.json'));
```

Comment:
@vpandiarajan20 Maybe we should only put this in the wrapper JS for now so we can get a proper review for the other content